supplements/tashas-cauldron-of-everything/archetypes/artificer-battlesmith.xml

```diff
@@ -0,0 +1,302 @@
+		<description>
+			<p>Melee Weapon Attack: your spell attack modifier to hit, reach 5 ft., one target you can see. Hit: 1d8 + PB force damage.</p>
+		</description>
+		<sheet>
+			<description>Melee Weapon Attack: your spell attack modifier to hit, reach 5 ft., one target you can see. Hit: 1d8 + PB force damage.</description>
+		</sheet>
+	</element>
+	<element name="Repair" type="Companion Action" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_COMPANION_ACTION_STEEL_DEFENDER_REPAIR">
+		<description>
+			<p>The magical mechanisms inside the defender restore 2d8 + PB hit points to itself or to one construct or object within 5 feet of it.</p>
+		</description>
+		<sheet usage="3/Day">
+			<description>The magical mechanisms inside the defender restore 2d8 + PB hit points to itself or to one construct or object within 5 feet of it.</description>
+		</sheet>
+	</element>
+	<element name="Deflect Attack" type="Companion Reaction" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_COMPANION_REACTION_STEEL_DEFENDER_DEFLECT_ATTACK">
+		<description>
+			<p>The defender imposes disadvantage on the attack roll of one creature it can see that is within 5 feet of it, provided the attack roll is against a creature other than the defender.</p>
+		</description>
+		<sheet>
+			<description>The defender imposes disadvantage on the attack roll of one creature it can see that is within 5 feet of it, provided the attack roll is against a creature other than the defender.</description>
+		</sheet>
+	</element>
+
+	<element name="Steel Defender" type="Information" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_INFORMATION_STEEL_DEFENDER">
+		<description>
+			<p style="margin-bottom:-5px"><em>Medium construct</em></p>
+			<h4 style="margin-top:5px" />
+			<ul class="unstyled">
+				<li><strong>Armor Class</strong> 15 (natural armor)</li>
+				<li><strong>Hit Points</strong> 2 + your Intelligence modifier + five times your artificer level (the defender has a number of Hit Dice [d8s] equal to your artificer level)</li>
+				<li><strong>Speed</strong> 40 ft.</li>
+			</ul>
+			<table style="text-align:center;margin-top:10px;margin-bottom:10px">
+				<thead><tr><td>STR</td><td>DEX</td><td>CON</td><td>INT</td><td>WIS</td><td>CHA</td></tr></thead>
+				<tr><td>14 (+2)</td><td>12 (+1)</td><td>14 (+2)</td><td>4 (-3)</td><td>10 (+0)</td><td>6 (-2)</td></tr>
+			</table>
+			<ul class="unstyled">
+				<li><strong>Saving Throws</strong> Dex +1 plus PB, Con +2 plus PB</li>
+				<li><strong>Skills</strong> Athletics +2 plus PB, Perception +0 plus PB x 2</li>
+				<li><strong>Damage Immunities</strong> poison</li>
+				<li><strong>Condition Immunities</strong> charmed, exhaustion, poisoned</li>
+				<li><strong>Senses</strong> darkvision 60 ft., passive Perception 10 + (PB x 2)</li>
+				<li><strong>Languages</strong> understands the languages you speak</li>
+				<li><strong>Challenge</strong> —</li>
```

Comment:
forgot to do this previously
```suggestion
				<li style="margin-bottom:-1em"><strong>Challenge</strong> —</li>
```

Reply:
Also can you create a copy of Information element and divs for ID into both Steel Defender feature and Subclass itself for the ERLW version, i think we should basically have them exact same in this regard.